devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -366,7 +367,7 @@ private boolean isIkea() {
 }
 
 private boolean isSomfySituo() {
-	device.getDataValue("model") == "Situo 4 Zigbee"
+	device.getDataValue("model") == "Situo 4 Zigbee" || device.getDataValue("model") == "Situo 1 Zigbee"
```

Comment:
Do you want to just refactor this to `isSomfy` and just check the manufacturer?

Reply:
Done

Reply:
I suppose naming the function `isSomfy` or leaving as `isSomfySituo` is fine, as they are both accurate.